### DISCUSSION TITLE
micro-service/nacos/Nacos%E5%8D%95%E6%9C%BA%E9%9B%86%E7%BE%A4%E9%83%A8%E7%BD%B2%E6%90%AD%E5%BB%BA

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#549] [Announcements]
# micro-service/nacos/Nacos%E5%8D%95%E6%9C%BA%E9%9B%86%E7%BE%A4%E9%83%A8%E7%BD%B2%E6%90%AD%E5%BB%BA

Nacos 单机&amp;集群部署搭建 本文介绍为Nacos2.2 版本配置，其他版本请注意根据情况而定 Nacos部署环境 Nacos定义为一个IDC内部应用组件，并非面向公网环境的产品，建议在内部隔离网络环境中部署，强烈不建议部署在公共网络环境。 Nacos部署模式 单机模式 - 用于测试和单机试用。 集群模式 - 用于生产环境，确保高可用。 多集群模式 - 用于多数据中心场景。

http://nas.edkso.cn:38080/micro-service/nacos/Nacos%E5%8D%95%E6%9C%BA%E9%9B%86%E7%BE%A4%E9%83%A8%E7%BD%B2%E6%90%AD%E5%BB%BA.html

**Comment by lincheng77** [comment]
111111
